mnist/README.md

```diff
@@ -469,6 +469,21 @@ There are various ways to monitor workflow/training job. In addition to using `k
 
 ### Tensorboard
 
+#### Local storage
```

Comment:
@eqqe Cloud you please update the Contents of Readme file to add the section? Thanks.